docs/software-platforms/aws/index.md

```diff
@@ -31,5 +31,5 @@ Please refer to these pages to learn more about setting up and using LSfR:
 - [Accessing a virtual computer](./accessing-vcs.md)
 - [Working with LSfR volumes](./working-with-volumes.md)
 - [Developing analyses on LSfR](./starting-development-on-lsfr.md)
-  - You can also find instructions for setting up virtual computers for development on this page.
+    - You can also find instructions for setting up your virtual computers for OpenScPCA development on this page.
```

Comment:
I don't think we need both changes... I also wonder if we want to clarify what we mean by "virtual computer" here. I have a different meaning in my head, so I think we should clarify that these are "in the cloud" somewhere in this section/title. 


```suggestion
    - You can also find instructions for setting up virtual computers for OpenScPCA development on this page.
```